aspnetcore/signalr/version-differences.md

```diff
@@ -86,6 +98,10 @@ npm install @aspnet/signalr
 
 The dependency on jQuery has been removed, however projects can still use jQuery.
 
+### Internet Explorer support
+
+Microsoft Internet Explorer 11 is the lowest supported version of IE.
```

Comment:
"is the lowest supported version of IE in ASP.NET Core SignalR" or something of the sort.